ts/parts/SvgRenderer.ts

```diff
@@ -83,6 +104,9 @@ declare global {
             style?: (string|CSSObject);
             translateX?: number;
             translateY?: number;
+            // width?: number;
+            // x?: number;
+            // y?: number;
```

Comment:
Are these future property types, when we migrated the usage from `number/string` to `number` only?

Reply:
I don't remember the context, but I vaguely remember wanting these properties there, but something failed.